client/ayon_core/hosts/max/api/pipeline.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-"""Pipeline tools for OpenPype Houdini integration."""
+"""Pipeline tools for AYON Houdini integration."""
```

Comment:
I know that this is changing OP->AYON, but this is so obviously wrong...

```suggestion
"""Pipeline tools for AYON 3ds max integration."""
```